common/types/nipost.go

```diff
@@ -1,8 +1,11 @@
 package types
 
+// TODO(mafa): remove these types after all is fully migrated to local DB
```

Comment:
TODO: consider moving them from here to `nipost_state` with the other deprecated functions